eras/byron/ledger/impl/cardano-ledger-byron.cabal

```diff
@@ -339,7 +339,7 @@ library testlib
     -Wunused-packages
 
   build-depends:
-    base >=4.14 && <5,
+    base >=4.18 && <5,
```

Comment:
This line can completely be deleted I suppose, since we are specifying the version number in the same file earlier?